### DISCUSSION TITLE
MPV懒人版如何调用n卡驱动级的自动HDR和SVR,而无需安装其他插件

### DISCUSSION THREAD

**Discussion by zong-yanyun** [hooke007/mpv_PlayKit#465] [mpv 通用]
事情是这样的，我有一个精简版的MPV播放器，是蜗牛云盘自带的，我惊奇的发现，它居然可以调用n卡驱动自带的SVR和自动HDR，这版本非常的精简，应该是没有安装多余的插件。请问我在MPV懒人版上可以实现驱动级的SVR吗，因为我有插帧和自动HDR的需求，想要尽可能的精简一些，但我的精简版不能插帧
我附上几张精简版MPV的代码截图
![屏幕截图 2024-05-23 124732](https://github.com/hooke007/MPV_lazy/assets/62926015/f093ae27-bc7f-477c-a663-b7ed50e1bced)
![屏幕截图 2024-05-23 124749](https://github.com/hooke007/MPV_lazy/assets/62926015/c5c9f5e9-b5c3-4f13-9679-033eb337c1d7)
![屏幕截图 2024-05-23 124838](https://github.com/hooke007/MPV_lazy/assets/62926015/4656380a-398b-4ed3-96de-997f5eba24a0)

**Comment by hooke007** [comment]
https://github.com/mpv-player/mpv/pull/14155
可能是自己凑出来的
我不会测试任何没进主线的东西

**Reply by zong-yanyun** [reply]
老哥，是直接参照上面链接就可以实现n卡驱动级SVR吗

**Reply by hooke007** [reply]
上面是mpv的pr

**Comment by hooke007** [comment]
重复 https://github.com/hooke007/MPV_lazy/discussions/286

**Reply by zong-yanyun** [reply]
我试了下用我说的精简版的exe配合你的插件，也能用，插帧，自动hdr，fsr同时能开，但是显卡占用率很高，它应该是把自动hdr的插件集成了，在exe里的，现在还是退回大佬的懒人版了，占用率只有之前的60%左右

**Reply by zong-yanyun** [reply]
请问下懒人版可以实现sdr实时转hdr吗，找了半天没找到。。。谢谢

**Reply by hooke007** [reply]
mpv 的sdr实时转hdr 自带，nvidia hdr也不在主线里，也不一定需要内部实现 https://github.com/mpv-player/mpv/issues/13352

另外 mpv的sdr2hdr 这里也有写 https://github.com/hooke007/MPV_lazy/wiki/0_FAQ#sdr%E5%AE%9E%E6%97%B6%E8%BD%AChdr%E7%9B%B4%E5%87%BA
